lib/scripts/repo_export_in_batches.rb

```diff
@@ -20,14 +22,22 @@
   raise(RegistryError, "Model (#{converter[:model_class].name}) for conversion must be an ActiveFedora::Base") unless converter[:model_class].ancestors.include?(ActiveFedora::Base)
 end
 
+puts "---------\nCreating Collection Store\n---------"
+collection_store = {}
+Collection.find_each do |collection|
```

Comment:
Are we going to build the `collection_store` for each record every time we run the export or does it make sense to memoize this information somehow? Since this is something that is tied to the `Collection` model maybe we can add a field to the `Collection` table in the database with a callback on record update to recalculate the collections_path. If that's too much we could also just use a memoized file like we've done with other data. Either way, I think that's a lot faster than recursively checking every collection for collections.

Reply:
Oh okay. Since it is run only once let's just leave it as is.